src/OrchardCore.Modules/OrchardCore.Themes/README.md

```diff
@@ -0,0 +1,247 @@
+# Orchard Core Theming explained
+
+This article explains how a Content Item is rendered and how the many ways the HTML that is rendered can be customized. It also explains the fundamental theming contepts that are __Shapes__, __Alternates__, __Templates__, __Differentiators__, __Content Zones__ and __Display Types__.
```

Comment:
I don't understand this sentence:
`how the many ways the HTML that is rendered can be customized`

Reply:
It is using some strong english grammatical knowledge ;) I understand it can be hard to understand, but I think it's totally valid english. Let's see what others have to say. I will rephrase to make it simpler though.